1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,24 @@
-function setAlarm() {}
+function formatCounter(counter) {
+    var minutes = Math.floor(counter / 60);
+    var seconds = counter % 60;
+    return minutes + ":" + seconds;   
+}
+
+function setAlarm() {
+  var setValue = document.getElementById("alarmSet").value;
+  var timeremainingEl = document.getElementById("timeRemaining");
```

Comment:
Use `const` or `let` instead of `var`: The var keyword has function scope, which can cause issues with variable hoisting and unexpected behavior. Use `const` or `let` instead to avoid these issues.